runtime/src/bank.rs

```diff
@@ -3902,7 +3905,12 @@ impl Bank {
                 .accounts()
                 .accounts_db
                 .has_accounts_update_notifier()
-                .then(|| sanitized_txs.iter().collect::<Vec<_>>());
+                .then(|| {
+                    sanitized_txs
+                        .iter()
+                        .map(|tx| tx.as_sanitized_transaction())
+                        .collect::<Vec<_>>()
+                });
```

Comment:
This change isn't necessary since `sanitized_txs` is still just a slice of `SanitizedTransaction`. Is there a place where the adapter is actually needed?

Reply:
Next PR these won't necessarily be SanitizedTransaction, but we still need the SanitizedTransaction for the sgeyser interface

Reply:
Can you make the change in the next PR then?

Reply:
The point of putting it here is to simplify review and make next PR smaller.

Pairing it here it's very obvious that it's no change in functionality

Reply:
I think you tried making the PRs too small

Reply:
Disagree. It's a separable, easily reviewable chunk of code.

I don't want it cluttered with 500  lines of type changes, and then it's not clear why this changed.

Reply:
The code changes here in bank.rs have no functional change and are just a few lines. Makes a lot more sense to change this when you change from a slice of `SanitizedTransaction` to using traits

Reply:
Agree there's no functional change. AND you could verify that in like 3s because this PR is small - so it's incredibly obvious.

Why are get hung up on the name change?

Reply:
I don't really understand your position either haha. I just don't really get how I'm supposed to review whether your new trait makes sense until you actually use it in a place where it's needed

Reply:
As discussed privately - will hold off on this PR until after the `RuntimeTransaction` transition (very soon). At that point this PR can actually change the fn(s) to use `RuntimeTransaction<impl SVMTransactionAdapter>`.